companies/123greetings.json

```diff
@@ -1,22 +1,23 @@
 {
-    "slug": "123greetings",
+    "slug": "123greeting2s",
```

Comment:
🚫 **[recordlint]** <[match-filename-slug](https://github.com/datenanfragen/data/#data-formats)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Filename `123greetings.json` does not match slug `123greeting2s`.

Either rename the file or change the slug.
```suggestion
    "slug": "123greetings",
```